projects/vdk-core/plugins/vdk-plugin-control-cli/src/vdk/plugin/control_cli_plugin/control_service_properties.py

```diff
@@ -105,10 +106,11 @@ def _new_http_session(self):
         session.auth = self._authenticator.authenticate()
         return session
 
-    def write_properties(self, job_name, properties):
+    def write_properties(self, job_name: str, team_name: str, properties: Dict):
         """
         We are adding retries on writing properties
         since we are using PUT to overwrite all properties which is idempotent operation.
+        :param team_name:
```

Comment:
Was this line left in accidentally? Not :param line for job_name or properties, and there is no explanation of the team_name parameter.

Reply:
```suggestion
```